packages/main/src/plugin/mcp/mcp-registry.ts

```diff
@@ -324,6 +324,7 @@ export class MCPRegistry {
     }
 
     const name = serverDetail.name;
+    const description = serverDetail.description;
```

Comment:
suggestion: avoid introducing const for name and description, as they are very generic names only used in the registerMCPClient call, it is possible to use directly serverDetail.name, and serverDetail.description

Reply:
moved to destruct